lib/elixir/src/elixir_tokenizer.erl

```diff
@@ -1455,7 +1455,7 @@ unexpected_token_or_reserved(_) -> "unexpected token: ".
 missing_terminator_hint(Start, End, #elixir_tokenizer{mismatch_hints=Hints}) ->
   case lists:keyfind(Start, 1, Hints) of
     {Start, {HintLine, _, _}, _} ->
-      io_lib:format("\n\n    HINT: it looks like the \"~ts\" on line ~B does not have a matching \"~ts\"",
+      io_lib:format("\nHINT: it looks like the \"~ts\" on line ~B does not have a matching \"~ts\"",
```

Comment:
I wonder if we should write them as `hint: ` now? so it matches the `error: ` notation? Before it made more sense to be in uppercase, since it could be multiple lines after. Thoughts?

Reply:
Should we update the other "HINT"s that were not using this indentation before to be lowercase as well?

I've also thought of applying an ANSI color like blue to them if enabled, it looks cool as well, it just needs some work of interpolating conditionally on ansi_enabled

Reply:
I checked and it seems these two are the last two hints? Adding ansi color sounds good to!

Reply:
types.ex has some examples of code still using `HINT`. I guess this is for the cases where we have a long error message and show it at the end.

Reply:
We can change the one in types too, right? Since it also includes the snippet?

Reply:
Yeah, it should be fine

Just added the blue ansi color text to the "hint:" and will update the other cases